agent/templates/index.html

```diff
@@ -1,37 +1 @@
-<html>
-<head>
-    <title></title>
-</head>
-<body>
-  <pre id="log"></pre>
-  <input type='button' value='전송' onclick='sendMsg();'>aaaa</input>
```

Comment:
Why do you use korean word as the name of value?

Reply:
This is the part of deleted lines in index.html.